parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnReaderImpl.java

```diff
@@ -352,7 +352,7 @@ public ColumnReaderImpl(ColumnDescriptor path, PageReader pageReader, PrimitiveC
       this.dictionary = null;
     }
     this.totalValueCount = pageReader.getTotalValueCount();
-    if (totalValueCount == 0) {
+    if (totalValueCount <= 0) {
       throw new ParquetDecodingException("totalValueCount == 0");
```

Comment:
I would also change the error message here to provide the value of totalValueCount (and change the equality)